src/cmd/manage_nsfs.js

```diff
@@ -301,13 +302,18 @@ async function delete_bucket(data) {
     const bucket_config_path = get_config_file_path(buckets_dir_path, data.name);
     try {
         const bucket_temp_dir_path = path.join(data.path, config.NSFS_TEMP_DIR_NAME + "_" + data._id);
-        await native_fs_utils.folder_delete(bucket_temp_dir_path, fs_context_fs_backend, true);
-        await native_fs_utils.delete_config_file(fs_context_config_root_backend, buckets_dir_path, bucket_config_path);
+        const entries = await nb_native().fs.readdir(fs_context_fs_backend, data.path);
+        const filter_entries = entries.filter(element => !element.name.endsWith(config.NSFS_TEMP_DIR_NAME + "_" + data._id));
```

Comment:
I think it will be more readable to change the variable name to a more meaningful name (then `filter_entries`).